multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/clients/CustomControllerClient.java

```diff
@@ -15,61 +14,38 @@
 
 public abstract class CustomControllerClient {
 
-    private final WebClientFactory webClientFactory;
+    protected final CloudControllerClient client;
+    private final WebClient webClient;
 
-    protected CustomControllerClient(WebClientFactory webClientFactory) {
-        this.webClientFactory = webClientFactory;
+    protected CustomControllerClient(CloudControllerClient client) {
+        this.client = client;
+        this.webClient = new WebClientFactory().getWebClient(client);
     }
 
-    protected WebClient getWebClient(CloudControllerClient client) {
-        return webClientFactory.getWebClient(client);
-    }
-
-    protected List<Map<String, Object>> getAllResources(WebClient webClient, String controllerUrl, String urlPath) {
-        return getAllResources(webClient, controllerUrl, urlPath, Collections.emptyMap());
-    }
-
-    protected List<Map<String, Object>> getAllResources(WebClient webClient, String controllerUrl, String urlPath,
-                                                        Map<String, Object> urlVariables) {
+    protected List<Map<String, Object>> getAllResources(String urlPath, Object... urlVariables) {
```

Comment:
Rename `urlPath` to `path` for consistency with `addPageOfResources`.

Reply:
Done